backend/src/apiserver/server/experiment_server_test.go

```diff
@@ -1089,42 +1089,43 @@ func TestArchiveAndUnarchiveExperiment(t *testing.T) {
 	clients, manager, experiment := initWithExperimentAndPipelineVersion(t)
 	defer clients.Close()
 	runServer := NewRunServer(manager, &RunServerOptions{CollectMetrics: false})
-	run1 := &apiV1beta1.Run{
+	run1 := &apiv1beta1.Run{
 		Name:               "run1",
 		ResourceReferences: validReferencesOfExperimentAndPipelineVersion,
 	}
-	err := runServer.validateCreateRunRequestV1(&apiV1beta1.CreateRunRequest{Run: run1})
-	assert.Nil(t, err)
-	_, err = runServer.CreateRunV1(nil, &apiV1beta1.CreateRunRequest{Run: run1})
+	// err := runServer.validateCreateRunRequestV1(&apiv1beta1.CreateRunRequest{Run: run1})
+	// assert.Nil(t, err)
```

Comment:
keep assert on validating request?

Reply:
The validation logic was moved inside the create functions. I removed the commented code.